plugins/ModuleLevelTrigger.hpp

```diff
@@ -137,11 +137,22 @@ class ModuleLevelTrigger : public dunedaq::appfwk::DAQModule
   int m_earliest_tc_index;
   int get_earliest_tc_index(const PendingTD& pending_td);
 
+  // Bitwords logic
+  bool m_use_bitwords;
+  nlohmann::json m_trigger_bitwords_json;
+  bool m_bitword_check;
+  std::bitset<16> m_TD_bitword;
+  std::vector<std::bitset<16>> m_trigger_bitwords;
+  std::bitset<16> get_TD_bitword(const PendingTD& ready_td);
```

Comment:
I'm wondering if the number 16 could be placed in some more more global variable so it's not hard-coded here (and many other places). Probably not configurable?

Reply:
As we discussed offline, this is not straightforward and would likely need dedicated spot somewhere outside daqconf. Let's keep it as is for now and revisit at later point.